src/Facebook/FacebookBatchResponse.php

```diff
@@ -102,7 +102,7 @@ public function addResponse($key, $response)
 
         $httpResponseBody = isset($response['body']) ? $response['body'] : null;
         $httpResponseCode = isset($response['code']) ? $response['code'] : null;
-        $httpResponseHeaders = isset($response['headers']) ? $response['headers'] : [];
+        $httpResponseHeaders = isset($response['headers']) ? $this->normalizeBatchHeaders($response['headers']) : [];
```

Comment:
useless private function, it just can be `array_column($response['headers'], 'value', 'name')` ;)